release

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env ruby
+require 'fog'
+require 'dotenv'
+require 'json'
+require 'digest'
+require 'set'
+require 'git'
+
+Dotenv.load
+
+ALL_RELEASES = "releases/all.json"
+LATEST_RELEASE = "releases/latest.json"
+GIT_DIR = File.expand_path(__FILE__ + "/..")
+
+class FileManager
+  attr_reader :bucket
+  def initialize(bucket)
+    @bucket = bucket
+  end
+
+  def upload!(filename, content)
+    file = bucket.files.new(
+      key: filename,
+      body: content,
+      public: true
+    )
+    file.save
+  end
+
+  def url(filename)
+    bucket.files.get(filename).public_url
+  end
+
+  def fetch(filename, default: "")
```

Comment:
We could move `ALL_RELEASES` and `LATEST_RELEASE` into this class if we provide a public method for getting each of these files.